lib/heapwatch.js

```diff
@@ -21,28 +21,31 @@ class HeapWatch {
         this.timeoutHandle = undefined;
         this.gcReportInterval = undefined;
         this.cumulativeGCTimes = Object.assign({}, ZERO_CUMULATIVE_GC_INTERVAL);
+        this.reportStatsHandler = (stats) => {
```

Comment:
Idem as for BaseService. Let's not hide method declarations like this.

Reply:
Idem to that answer in the base_service.